Optik/Classes/AlbumViewController.swift

```diff
@@ -84,12 +85,14 @@ internal final class AlbumViewController: UIViewController {
         self.dismissButtonPosition = dismissButtonPosition
         
         pageViewController = UIPageViewController(transitionStyle: .scroll,
-                                                  navigationOrientation: .horizontal,
+                                                  navigationOrientation: .vertical,
                                                   options: [UIPageViewControllerOptionInterPageSpacingKey : Constants.SpacingBetweenImages])
-
+        
+        pageControl = UIPageControl()
```

Comment:
why isn't that in one of the setup methods? You should respect the current architecture when you add things!